src/service.ts

```diff
@@ -300,11 +301,15 @@ export class Service {
     if (!result.success) {
       throw new Error((result as any).message);
     }
-    let data = decodeRestrictedBase64ToBytes(result.output);
-    if (isZlibData(data)) {
-      data = await decompressZlib(data);
+    let wasm = decodeRestrictedBase64ToBytes(result.output);
+    if (isZlibData(wasm)) {
+      wasm = await decompressZlib(wasm);
     }
-    return data;
+    const ret: any = {wasm};
+    if (result.wasmBindgenJs) {
+      ret.wasmBindgenJs = result.wasmBindgenJs;
+    }
+    return ret;
```

Comment:
Let's not change the signature of the `compileFile` (to avoid breaking old fiddles -- @mbebenita will be upset about his article not working :) ). Add `compileFileWithBindings`: you can copy compileFile and/or their extract common parts into private methods. Thanks